### ISSUE TITLE
Double entries in parallels in csv files

### ISSUE THREAD

**Issue by ayya-vimala** [ayya-vimala/pali-networks#34]
#### Bug Description
The same text sometimes appears twice or more times in text. The system now picks that up as two separate parallels to a certain segment. However, this gives a distorted picture. It would be better that only the first entry was mentioned, maybe with a marker to show there are multiple entries in this specific file.

#### Expected behaviour
Only the first entry of that file to be mentioned for each segment of the source file.

**Comment by sebastian-nehrdich** [comment]
As we made some recent changes to the JSON files (introducing the offset values), it would make sense to rethink how we can approach the case of longer segments where two different quotes appear in the same segment. This is not so much an issue for the Sanskrit and Tibetan data that we worked with up to now, but it would make sense to think about how we can solve this issue. I have no idea currently on what would be a good way to deal with these cases, but we should keep in mind that we will run into this problem at some point!
